### ISSUE TITLE
local: Fix characters allowed in filenames on windows

### ISSUE THREAD

**Issue by ncw** [ncw/rclone#161]
The current list of characters not allowed on windows are these - see [local.go cleanUtf8](https://github.com/ncw/rclone/blob/master/local/local.go#L179)

    '<', '>', '"', '|', '?', '*', '&'

However this doesn't quite agree with the [ms official documentation](https://msdn.microsoft.com/en-us/library/aa365247)

Which does allow '&' in file names and doesn't allow ':'.

**Comment by klauspost** [comment]
I think the issue was that '&' is a special character in the shell.

**Comment by ncw** [comment]
Thanks @klauspost that was super speedy!

**Comment by klauspost** [comment]
Oh - now I remember why colon was not replaced:

```
>rclone.exe copy f:\up\300-399 od:/delme
2015/10/12 19:41:17 Local file system at \\?\d:\dev\gopath\src\github.com\ncw\rclone\f_\up\300-399: Failed to
open directory: \\?\d:\dev\gopath\src\github.com\ncw\rclone\f_\up\300-399: GetFileAttributesEx \\?\d:\dev\gopath\src\github.com\ncw\rclone\f_\up\300-399: The system cannot find the path specified.
```

**Comment by ncw** [comment]
@klauspost so we need to preserve `:` when it is used as a drive letter - is that right?

**Comment by ncw** [comment]
@klauspost actually I think you fixed this in 16c9fba5dedb62c72ddc6f631b726d7cb18b83cd already so I'm going to close this issue.  If this is wrong then please re-open!

Thanks

Nick
